README.md

```diff
@@ -60,48 +58,26 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [To Do list: interactive list] <a name="about-project"></a>
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to change this title also so your README file uses the name of this project
```suggestion
# 📖 [Leaderboard] <a name="about-project"></a>
```